### PR TITLE
feat(examples): updates to hello-world example and directory reorganization

### PR DESCRIPTION
**Pull request by campionfellin**
We want more examples (as per https://github.com/awslabs/cdk8s/issues/22) but we should improve the current examples first so they work out of the box (`git clone` straight to example).

🎉First PR 🎉

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by campionfellin** [comment]
Looks like the `yarn build` from Lerna is failing because the `build` script in the example doesn't do the generate first.

Do we want:

1) Lerna to ignore the `/examples`
2) Examples to contain the `cdk8s generate` in build script?

I'd lean more towards 1, as I don't think `cdk8s generate` is needed for every build.

**Comment by eladb** [comment]
@campionfellin I think we should reorganize `examples`  such that each example is a separate project (like yours) and change lerna and yarn from looking under `examples` to `examples/*` (should be both in `package.json` and `lerna.json`).

File: examples/podinfo/examples/app-example.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ class MyChart extends Chart {
       faultError: true
     });
 
-    const deployment = new Deployment(this, 'Deployment', {
+    const deployment = new Deployment(this, 'deployment', {
```
Comment (campionfellin) [inline examples/podinfo/examples/app-example.ts]: I found that upper case letters were not allowed:

```
throw new Error(`"${comp}" is not a valid object name. The characters allowed in names are: digits (0-9), lower case letters (a-z), -, and .`);
                ^

Error: "Deployment" is not a valid object name. The characters allowed in names are: digits (0-9), lower case letters (a-z), -, and .
```

File: examples/podinfo/examples/podinfo.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { Deployment, ImagePullPolicy, IContainer } from "../lib";
-import { Container } from '../../imports/k8s';
+import { Container } from '../.gen/apps-deployment-v1'
```
Comment (campionfellin) [inline examples/podinfo/examples/podinfo.ts]: New way of importing was not working for me. Does https://github.com/awslabs/cdk8s/commit/2db4cfbb20e7b6d8b09e8b8c9e6d096f0ea9fefe need to be published?

Comment (eladb) [reply examples/podinfo/examples/podinfo.ts]: Yes, try to install the @next dist tag:

“yarn global add cdk8s-cli@next”
“yarn add cdk8s@next”

Comment (campionfellin) [reply examples/podinfo/examples/podinfo.ts]: Awesome, that worked!

**Pull request merged**